recipes/sextractor/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+mv man/sex.1.in man/s_extractor.1.in
+mv man/sex.x man/s_extractor.x
```

Comment:
Here they need to be changed too.

Reply:
Yep; I missed it in a couple other places too.